pkg/k8shandler/status.go

```diff
@@ -49,3 +54,35 @@ func updateNodeStatus(node *nodeState, dpl *v1alpha1.ElasticsearchStatus) {
 	}
 	dpl.Nodes = append(dpl.Nodes, nodeStatus)
 }
+
+func clusterHealth() string {
+	config := &ExecConfig{
+		namespace:      "openshift-logging",
+		labelName:      "component",
+		labelPrefix:    "elasticsearch",
+		containerName:  "elasticsearch",
+		command:        []string{"es_util", "--query=_cluster/health?pretty=true"},
+		kubeConfigPath: "/home/origin/admin.kubeconfig",
```

Comment:
Is this a correct path?

Reply:
those are just placeholders, I'm still searching for the correct source of these two variables

Reply:
the kube config path should be made available to the operator already. We can override it with an env var, so it stores it somewhere.